### DISCUSSION TITLE
add back hit animation for osu!taiko when using hidden mod just like when hidden is not applied

### DISCUSSION THREAD

**Discussion by aphelionosu** [ppy/osu#22050] [Ideas]
IDK I feel like this would be good if this added back (maybe as ruleset option) as some players like me feels more comfortable if hit animation still there even when hidden mod is used (something like image below)
![image](https://user-images.githubusercontent.com/41836319/211056988-3a4950e2-4779-4d10-b03f-815e100db79c.png)

**Comment by peppy** [comment]
Confused as to what you want. Can you show this with two videos comparing?
